src/main/java/com/faforever/client/vault/replay/LiveReplayController.java

```diff
@@ -43,25 +46,35 @@
   private final UiService uiService;
   private final I18n i18n;
   private final MapService mapService;
+  private final TimeService timeService;
   public TableView<Game> liveReplayControllerRoot;
   public TableColumn<Game, Image> mapPreviewColumn;
+  public TableColumn<Game, Instant> startTimeColumn;
   public TableColumn<Game, String> gameTitleColumn;
   public TableColumn<Game, Number> playersColumn;
   public TableColumn<Game, String> modsColumn;
   public TableColumn<Game, String> hostColumn;
   public TableColumn<Game, Game> watchColumn;
 
-  public LiveReplayController(GameService gameService, UiService uiService, I18n i18n, MapService mapService) {
+  public LiveReplayController(GameService gameService, UiService uiService, I18n i18n,
+                              MapService mapService, TimeService timeService) {
     this.gameService = gameService;
     this.uiService = uiService;
     this.i18n = i18n;
     this.mapService = mapService;
+    this.timeService = timeService;
 
     selectedGame = new SimpleObjectProperty<>();
   }
 
   public void initialize() {
-    initializeGameTable(gameService.getGames().filtered(game -> game.getStatus() == GameStatus.PLAYING));
+    Comparator gameSorter = Comparator.comparing(
+        (Game game) -> game.getStartTime(),
+        Comparator.nullsFirst(Comparator.naturalOrder()))
+        .reversed();
+    initializeGameTable(gameService.getGames()
+                    .filtered(game -> game.getStatus() == GameStatus.PLAYING)
+                    .sorted(gameSorter));
```

Comment:
This passes a `SortedList` as an `ObservableList` which is then wrapped into a `SortedList`. I doubt that this is the way to go, and IntellIJ even warns you about it :(
There's already `liveReplayControllerRoot.comparatorProperty()`.

Not sure what's the side effect is but this needs to be cleaned up.

Reply:
@Wesmania use Intelij :D